src/MergeContext.js

```diff
@@ -235,12 +237,18 @@ class MergeContext {
         return false;
     }
 
+    async _tagCommit() {
+        if (this._tagCommitCache)
+            return this._tagCommitCache;
+         this._tagCommitCache = await GH.getCommit(this._tagSha);
```

Comment:
You can simplify by making the assignment (i.e., cache initialization) conditional instead of returning twice.